File: aula8/exercicio_2/python_logic/exercicio2_modular.py
def calcular_area_retangulo(base, altura):
    # Aqui a função devolve o valor calculado para o programa principal
    return base * altura

base = float(input("Digite a base do retângulo: "))
altura = float(input("Digite a altura do retângulo: "))
area = calcular_area_retangulo(base, altura)
print(f"Área do retângulo: {area}")

File: aula8/exercicio_2/cpp_logic/exercicio2_modular.cpp
#include <iostream>
using namespace std;

double calcularAreaRetangulo(double base, double altura) {
    // Aqui a função retorna um valor numérico para o main usar depois
    return base * altura;
}

int main() {
    double base, altura, area;


    cout << "Digite a base do retangulo: ";
    cin >> base;
    cout << "Digite a altura do retangulo: ";
    cin >> altura;
    area = calcularAreaRetangulo(base, altura);
    cout << "Area do retangulo: " << area << endl;

    return 0;
}